addons/ui/functions/fnc_checkWeaponZeroing.sqf

```diff
@@ -0,0 +1,27 @@
+#include "..\script_component.hpp"
+/*
+ * Author: drofseh
+ * Display the zeroing of the player's current muzzle.
+ *
+ * Arguments:
+ * 0: Unit <OBJECT>
+ *
+ * Return Value:
+ * Current Zero <NUMBER>
+ *
+ * Example:
+ * [ace_player] call ace_ui_fnc_checkWeaponZeroing
+ *
+ * Public: No
+ */
+
+params ["_unit"];
+
+[
+    [LLSTRING(weaponZeroSetTo)],
+    [format ["%1 %2", str currentZeroing _unit, LLSTRING(meters)]],
+    true
+] call CBA_fnc_notify;
+
+_currentZero
```

Comment:
Can just show the text from the display
this handles zeros set from rangefinder

```suggestion
private _d = ((uiNamespace getVariable "IGUI_displays") + [findDisplay 46]) select {300 == ctrlIDD _x}; 
private _c = (_d param [0, displayNull]) displayCtrl 168;

[
    [LLSTRING(weaponZeroSetTo)],
    ctrlText _c,
    true
] call CBA_fnc_notify;
```

Reply:
Sounds great to me!

Reply:
I mentioned this in discord but want to put it here too.
Getting into the drivers seat of the vehicle stops this from working.